invenio_app_ils/documents/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -259,9 +259,7 @@
           "type": "object"
         },
         "extensions": {
-          "type": "object",
-          "dynamic": false,
```

Comment:
I would probably have left `dynamic: false`... any particular reason why you have removed it?

Reply:
If we keep `dynamic: false` we cannot search for the extensions in the search-box. I didn't check further.